app/messages/users_list_message.rb

```diff
@@ -4,26 +4,35 @@ module VCAP::CloudController
   class UsersListMessage < MetadataListMessage
     register_allowed_keys [
       :usernames,
+      :partial_usernames,
```

Comment:
This should also be added to the API documentation of the following endpoints:

- /v3/users
- /v3/organizations/:guid/users
- /v3/spaces/:guid/users